checkov/helm/runner.py

```diff
@@ -25,14 +25,13 @@
 
 
 class K8sHelmRunner(k8_runner):
-    check_type = CheckType.HELM
-
     def __init__(self, graph_class: Type[LocalGraph] = KubernetesLocalGraph,
                  db_connector: NetworkxConnector = NetworkxConnector(),
                  source: str = "Kubernetes",
                  graph_manager: Optional[GraphManager] = None,
                  external_registries: Optional[List[BaseRegistry]] = None) -> None:
         super().__init__(graph_class, db_connector, source, graph_manager, external_registries)
+        self.check_type = CheckType.HELM
```

Comment:
this attribute cannot be static?

Reply:
it's not static in all the other runners